File: STALK/Assets/SCRIPTS/PLAYER/Camera.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class Camera : MonoBehaviour {

    public GameObject Player;
    public GameObject reference;
    private Vector3 distance;

    private void Start()
    {
        distance = transform.position - Player.transform.position;
    }

    private void LateUpdate()
    {
        distance = Quaternion.AngleAxis(Input.GetAxis("Mouse X") * 2, Vector3.up) * distance;

        transform.position = Player.transform.position + distance;
        transform.LookAt(Player.transform.position);

        Vector3 copiaRotation = new Vector3(0, transform.eulerAngles.y, 0);
        reference.transform.eulerAngles = copiaRotation;
    }
}

File: STALK/Assets/SCRIPTS/PLAYER/camera.js
var cam : Camera;
var cam1 :Camera;
var cam2 :Camera;
var cam3 :Camera;


function Start () {
	cam.enabled=true;
	cam1.enabled=false;
	cam2.enabled=false;
	cam3.enabled=false;
}

function Update () {
	if (Input.GetKeyDown("1")){
		cam.enabled=true;
		cam1.enabled=false;
		cam2.enabled=false;
		cam3.enabled=false;
	}
	if (Input.GetKeyDown("2")){
		cam.enabled=false;
		cam1.enabled=true;
		cam2.enabled=false;
		cam3.enabled=false;
	}
	if (Input.GetKeyDown("3")){
		cam.enabled=false;
		cam1.enabled=false;
		cam2.enabled=true;
		cam3.enabled=false;
	}
	if (Input.GetKeyDown("4")){
		cam.enabled=false;
		cam1.enabled=false;
		cam2.enabled=false;
		cam3.enabled=true;
	}
}
